demo/build.gradle

```diff
@@ -68,55 +54,21 @@ dependencies {
     implementation 'androidx.lifecycle:lifecycle-extensions:2.2.0'
 
     // GMS
-    gmsImplementation 'com.google.android.gms:play-services-maps:17.0.1'
-    gmsImplementation project(':library')
+    implementation 'com.google.android.gms:play-services-maps:17.0.1'
+    implementation project(':library')
 
-    // V3
-    v3Implementation 'com.google.android.libraries.maps:maps:3.1.0-beta'
-    v3Implementation 'com.android.volley:volley:1.2.1' // TODO - Remove this after Maps SDK v3 beta includes Volley versions on Maven Central
-    v3Implementation project(':library-v3')
-    v3Implementation 'androidx.multidex:multidex:2.0.1'
-    implementation "androidx.lifecycle:lifecycle-viewmodel-ktx:2.5.1"
```

Comment:
L80 is included in the library, and L79 is a sub-dependency from the Play Services Map and the Compatibility package.

On a second thought, it will be a better practice to lock on these versions, instead of relying on the version other libraries use to include them. I have re-added them.